examples/src/main/scala/example/stitching/ExampleApp.scala

```diff
@@ -111,7 +111,7 @@ object ExampleApp extends App {
                        .start(
                          8088,
                          Http.route {
-                           case _ -> Root / "api" / "graphql" => ZHttpAdapter.makeHttpService(interpreter)
+                           case _ -> Root / "api" / "graphql" => ZHttpAdapter.makeHttpService[ZEnv, CalibanError](interpreter)
```

Comment:
So downside of this change is that the effect invariance of the `ServerOptions` makes the ergonomics of the `ZHttpAdapter` _worse_ because you need to fully specify the types now. I'm trying to see if there is a way to work around it, like moving the server options into a separate parameter group, or using a partially applied pattern.